source/diagnostics.tex

```diff
@@ -1192,7 +1201,6 @@
 
   // \ref{syserr.errcode.nonmembers} non-member functions:
   error_code make_error_code(errc e) noexcept;
-  bool operator<(const error_code& lhs, const error_code& rhs) noexcept;
 
   template <class charT, class traits>
     basic_ostream<charT, traits>&
```

Comment:
I'm pretty sure I did mean to, because I moved it to line 894.

We could have it in both places (as we do for `make_error_code` and `operator<<`), but if we do that then we should also have `operator==` and the other overloads in both places. IMHO it makes no sense for _only_ `operator<` to be in the class synopsis.